frontend/app/model/RichContent.scala

```diff
@@ -9,10 +9,10 @@ abstract class RichContent(content: Content) {
 case class ContentItem(content: Content) extends RichContent(content)
 
 case class ContentItemOffer(content: Content) extends RichContent(content) {
-  val tagTitleOpt = content.tags.find { tag =>
+  val tagTitleOpt = content.tags.map { tag =>
     tag.id.toLowerCase match {
-      case "membership/membership-offers" | "membership/membership-competitions" => true
-      case _ => false
+      case "membership/membership-offers" | "membership/membership-competitions" => Some(tag.webTitle.replace("Membership ", ""))
```

Comment:
... whereas, might be nice to have a comment here explaining why we need to do this?